R/run_model.R

```diff
@@ -34,7 +34,12 @@ run_model <- function(data, options) {
          spectrum_path = system_file("output", "malawi_spectrum_download.zip"),
          summary_path = system_file("output", "malawi_summary_download.zip")))
   }
-  naomi::hintr_run_model(data, options)
+  dir <- normalizePath(dir, mustWork = TRUE)
+  output_path <- tempfile(tmpdir = dir, fileext = ".RDS")
```

Comment:
why are we using shouty case here? `.rds` is more typical

Reply:
No good reason - I have fixed that.